lib/engine/corporation.rb

```diff
@@ -25,13 +25,14 @@ class Corporation
     include Transfer
 
     attr_accessor :ipoed, :par_via_exchange, :max_ownership_percent, :float_percent, :capitalization, :max_share_price
-    attr_reader :companies, :min_price, :name, :full_name, :fraction_shares, :type
+    attr_reader :companies, :min_price, :name, :full_name, :fraction_shares, :type, :id
```

Comment:
While in the middle of the merger, the system and the original corporation have the same name. Allow id to be different than name.

Reply:
what happens if you change corporation id to be sym?

Reply:
Like name.to_sym? That should work fine. I can test it.

Reply:
asking about using sym as the id. 

can you just make a fake name instead? which is like name + ' system'

Reply:
That's what I'm doing. This is to get around the fact that sym is also @name, which is displayed on charters. I split them into different variables (line 35) and then overwrite the id to something unique for the system.